apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts

```diff
@@ -258,7 +258,7 @@ export function useDeepLinkHandler() {
           break;
         }
         case "discover":
-          if (path.startsWith("protect")) {
+          if (path?.startsWith("protect")) {
```

Comment:
apparently we have this real bug in `develop` btw, `ledgerlive://discover` will make a crash at boot with "startsWith" error, after i fix it the test pass, it was a great way for me to test but i think there is a bug on develop🥵 

cc @Justkant @ggilchrist-ledger

Reply:
Nice catch, I think typescript should catch that no ?